uprotocol/uattributes.proto

```diff
@@ -60,19 +60,20 @@ message UAttributes {
     // https://github.com/eclipse-uprotocol/uprotocol-spec/blob/main/basics/qos.adoc[uProtocol Prioritization]
     UPriority priority = 5;
 
-    // TTL is How long this message should live for after it was generated (in milliseconds).
+    // TTL is How long this message should live for after it was generated (in milliseconds). If not present,
+    // the message is assumed to not timeout (should live forever)
```

Comment:
In the code I see checks for `> 0` as well, which seems to suggest that we could/should rephrase this to
> If not present or 0 (zero), the message is assumed to not time out, i.e. should live forever.

Reply:
fixed thank you